docs/Makefile

```diff
@@ -2,7 +2,7 @@
 #
 
 # You can set these variables from the command line.
-SPHINXOPTS    = -W  # turn warnings into errors
+SPHINXOPTS    = # -W  # turn warnings into errors
```

Comment:
The new sphinx  version generates a bunch of new warnings. If that's OK I'll fix these in another PR to keep this one self contained.